lms/static/sass/shared-v2/_footer.scss

```diff
@@ -5,7 +5,7 @@
     @include clearfix();
     @extend %ui-print-excluded;
     box-shadow: 0 -1px 5px 0 $shadow-l1;
-    border-top: 1px solid tint(palette(grayscale, light), 50%);
+    border-top: 1px solid tint(palette(grayscale, back), 50%);
```

Comment:
Can we use one of the five values of gray? I'm hesitant to do this kind of thing because then we'll end up with our same fifty shades of grey. It looks like you're trying to lighten our base grey a smidge? What about using the `accent`, or `back`?